package.json

```diff
@@ -63,8 +63,8 @@
   "license": "MIT",
   "readmeFilename": "README.md",
   "peerDependencies": {
-    "react": ">= 16.6.3",
-    "react-native": ">= 0.58.2"
+    "react": "*",
+    "react-native": "*"
```

Comment:
wonder if this is going to come back to bite us if we don't at least pin to major version, is there a way to do like `16.*`?

Reply:
The error when installing only complains about the version of react native

```
npm ERR! Conflicting peer dependency: react-native@0.73.6
npm ERR! node_modules/react-native
npm ERR!   peer react-native@">= 0.58.2" from react-native-purchases@7.24.0
npm ERR!   node_modules/react-native-purchases
npm ERR!     react-native-purchases@"RevenueCat/react-native-purchases#tvos-deps" from the root project
```

So I am actually going to revert the react dependency change and only change `react-native` to `*`. `0.58` is so old it doesn't even show up in https://reactnative.dev/versions lol. `react-native-purchases-ui` is already pointing to `*`. I am confident this change is fine.